packages/muelu/src/Utils/MueLu_Monitor.hpp

```diff
@@ -61,6 +61,18 @@
 
 namespace MueLu {
 
+  struct Helper {
+    //! Helper function for object label
+    static std::string getLabel(const Level & level) {
```

Comment:
Maybe rename to `getPostfixedLabel`?

Reply:
Actually, maybe make a generic function (without label), which would do:
```c++
static std::string getPostfixedLable(const std::string& label) {
  if (label != "")
    return label + ": ";
  return label;
}
```
No need to make it `Level` specific.